draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -931,10 +955,25 @@ TBD
     <author fullname="Brent Zundel">
       <organization>Evernym</organization>
     </author>
-    <author fullname="David Chadwick">
-      <organization>University of Kent</organization>
+    <author fullname="Kyle Den Hartog">
+      <organization>MATTR</organization>
+    </author>
+    <date day="03" month="Mar" year="2022" />
+  </front>
+</reference>
+
+@VC_JWT
+
+<reference anchor="VC_JWT" target="https://w3c.github.io/vc-jwt/">
```

Comment:
Beware that this will change to https://www.w3.org/TR/vc-jwt when it goes to FPWD.

You might want to file an issue to track fixing it if you fear you will forget.

Reply:
thanks, issue #237